ouroboros-consensus-protocol/src/Ouroboros/Consensus/Protocol/Ledger/HotKey.hs

```diff
@@ -8,7 +8,7 @@
 -- | Hot key
 --
 -- Intended for qualified import
-module Ouroboros.Consensus.Shelley.Protocol.HotKey (
+module Ouroboros.Consensus.Protocol.Ledger.HotKey (
```

Comment:
Thinking out loud: is the hotkey more a `Ledger` concern or more a `BlockForging` concern?

Actually, I'm less sure how to distinguish which parts of this are unique to `BlockForging` than I had thought I was. EG only a block-producing node would ever use `evolve`, `sign`, and `mkHotKey`, if I understand correctly.

I suppose it's natural to leave this module unbroken, and so `Ledger` really is the most appropriate point in the conceptual hierarchy for this bag of definitions.